src/components/browse/dataset/datasetAccordion/datasetSummary/datasetSamples.tsx

```diff
@@ -25,18 +25,35 @@ const DatasetSamples = (props: dataSetSamplesProps) => {
           {props.samples !== null ? (
             <div>
               <div className="mb-0">
-                <strong>{props.samples.count}</strong>&nbsp;Samples (Sex:{" "}
-                {props.samples.stats?.sex?.map((x) => (
+                <strong>{props.samples.count}</strong>&nbsp;Samples{" "}
+                {props.samples.stats?.sex.length > 0 ? (
                   <>
-                    <span title={x.value}>{x.count + " " + x.value}</span>
-                    {", "}
+                    (Sex:&nbsp;
+                    {props.samples.stats?.sex?.map((x, idx, arr) => {
+                      let sex: string = "";
+                      if (x.value === "FEMALE_SEX_FOR_CLINICAL_USE") {
+                        sex = "Female";
+                      } else sex = "Male";
+                      let sep = ". ";
+                      if (idx === arr.length - 1) {
+                        sep = "";
+                      }
+                      return (
+                        <span title={sex} key={sex}>
+                          {x.count + " " + sex}
+                          {sep}
+                        </span>
+                      );
+                    })}
+                    )
                   </>
-                ))}
-                )
+                ) : (
+                  <></>
+                )}
                 <div className="my-0">
                   <p className="mb-0">
-                    <strong>{props.samples.stats?.tissues.length}</strong>{" "}
-                    &nbsp;Tissues:
+                    <strong>{props.samples.stats?.tissues.length}</strong>
```

Comment:
I don't see why, since the tissues field should always be present?

Reply:
If you assume it could be null, then it should be changed in the interface. Also it should be `tissues?.length || 0` then.

But I think the interface is right, metldata always provides the field, though it can be an empty array.